src/main/java/com/google/impactdashboard/server/api_utilities/IamBindingRetriever.java

```diff
@@ -109,20 +118,126 @@ public static IamBindingRetriever create() throws IOException, GeneralSecurityEx
     return membersforRoles;
   }
 
+  /**
+   * Returns all organization ids associated with organization-level roles in 
+   * the map of roles to members. 
+   */
+  private List<String> getUnknownOrganizationIds(Map<String, Integer> membersForRoles) {
+    return membersForRoles.keySet().stream()
+        .filter(role ->
+            !roles.stream().reduce(false, (accumulator, knownRole) ->
+                accumulator || role.equals(knownRole.getName()), Boolean::logicalOr))
+        .map(role -> {
+          List<String> items = Arrays.asList(role.split("\\s*/\\s*"));
+          if (!items.get(0).equals("organizations")) {
+            return "";
```

Comment:
I think in general, Pattern and Matcher are better approaches than this, but not required at this stage.